assemblies/assembly-disabling-telemetry-data-collection.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="assembly-disabling-telemetry-data-collection"]
```

Comment:
According to [redhat-documentation.github.io/modular-docs#module_anchor-and-file-names-concept](https://redhat-documentation.github.io/modular-docs/#module_anchor-and-file-names-concept), the file name is correct, but the id (or anchor) should be

> At the top of every module, provide an anchor in the format [id="filename_{context}"] where filename is the exact name of the file, without the file extension (.adoc) and a module prefix.

therefore:

```suggestion
[id="disabling-telemetry-data-collection_{context}"]
```